osbs/core.py

```diff
@@ -175,16 +171,17 @@ def logs(self, build_id, follow=False, namespace=DEFAULT_NAMESPACE):
         if build_json['status'].lower() in BUILD_PENDING_STATES:
             self.wait_for_build_to_get_scheduled(build_id, namespace)
 
-        # 0.4.3+
-        proxy_url = self._build_url("proxy/buildLogs/%s/?follow=%d" % (
-            build_id, 1 if follow else 0), namespace=namespace)
-        response = self._get(proxy_url, stream=follow, headers={'Connection': 'close'})
+        # 0.5+
+        buildlogs_url = self._build_url("buildLogs/%s/" % build_id,
+                                        follow=(1 if follow else 0),
+                                        namespace=namespace)
+        response = self._get(buildlogs_url, stream=follow, headers={'Connection': 'close'})
         if response.status_code in (403, 404):
-            # 0.4.1
-            # FIXME: remove this once 0.4.3 is deployed everywhere
-            buildlogs_url = self._build_k8s_url(
-                "containerLogs/default/build-%s/custom-build?follow=%d" % (
-                    build_id, 1 if follow else 0))
+            # 0.4.3
+            # FIXME: remove this once 0.5.? is deployed everywhere
+            buildlogs_url = self._build_url("proxy/buildLogs/%s/" % build_id,
+                                            follow=(1 if follow else 0),
+                                            namespace=namespace)
             response.close_multi()
```

Comment:
The problem was, that curl handle stayed open on the multi request and subsequent request raised an exception. I guess we could implement it as context manager.